projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/listeners/module/ModuleVersionHelper.java

```diff
@@ -84,14 +91,20 @@ public void createFunction(Module m, long[] args) {
 
         @Override
         public void createGlobalVariable(Module m, long[] args) {
-            int i = 0;
-            Type type = new PointerType(m.types.get(args[i++]));
-            boolean isConstant = (args[i++] & 1) == 1;
-            int initialiser = (int) args[i++];
-            long linkage = args[i++];
-            int align = (int) args[i];
-
-            m.generator.createGlobal(type, isConstant, initialiser, align, linkage);
+            // Checkstyle: stop magic number name check
+            Type type = new PointerType(m.types.get(args[0]));
+            boolean isConstant = (args[1] & 1) == 1;
+            int initialiser = (int) args[2];
+            long linkage = args[3];
+            int align = (int) args[4];
```

Comment:
Because I skip one of the read args (the one seems not to be important for us now), I used a constant instead of the incrementing variable to access the option.

I changed the remaining part of the method to the same style, because I think it is easier to read and change in the future than incrementing a variable. All those options are position dependent, so there is no need for such a variable.